### DISCUSSION TITLE
About cgroup memory usage on pods that runs java with JVM

### DISCUSSION THREAD

**Discussion by ldominguezBold** [netdata/netdata#14619] [Q&A] [answered]
Hi, I have netdata installed with helm into my k8s cluster, then i have some deploys that use Java, how to manage the alerts of high memory usage on cgroups because jvm using almost the pod ram limit, but really is not high because is the memory that get the JVM as a room for the Java process itself.... for example using JMX I can see that my program is using only 100MB of 2 GB of Xmx of JVM... The Resource Limit for that Pod is of 2GB and the JVM has a room of 1.8GB of RAM, so the JVM when start consume that 1.8GB of RAM and is triggering netdata alert about this ussage.

**Comment by ilyam8** [comment, marked as answer]
Hello. I think the only option is to turn off memory usage alerts for these Pods. Do these Pods have a common part in the name? If so, we could use [charts](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-charts).
